src/main/java/org/jahia/modules/sam/healthcheck/probes/SearchIndexProbe.java

```diff
@@ -65,7 +65,7 @@ public ProbeStatus getStatus() {
         if (queryAVG > queryAVGLastMinuteYellowThreshold) {
```

Comment:
check should also probably be reversed (check for red first then yellow)? otherwise I think will always be yellow, never red.

Also maybe check for NaN and flag as red as well